pallets/gear/src/queue.rs

```diff
@@ -182,7 +182,7 @@ where
             (context, code, balance).into(),
             (random.encode(), bn.unique_saturated_into()),
         )
-        .unwrap_or_else(|e| unreachable!("core-processor logic invalidated: {e}"))
+        .unwrap_or_else(|e| unreachable!("{e}"))
```

Comment:
NOTE: was incorrect info message, because errors can appear in case `pallet-gear` provides for `core-processor` incorrect arguments or suitable environment for execution has not been provided.